### DISCUSSION TITLE
what kind of service is used for the app

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#236] [Q&A] [answered]
I tried to find out what the status of the photobooth service is, but "systemctl status photobooth" or "systemctl status photobooth-app "
did find anything.

So what kind of service is used ?

**Comment by mgineer85** [comment]
Add —user after systemctl

**Reply by mgineer85** [reply, marked as answer]
https://photobooth-app.org/setup/installation/#setup-system-service

**Reply by Striker62** [reply]
thanks now it work
